### PR TITLE
Drawer Navigator Implementation

### PR DESCRIPTION
**Pull request by ESmike0**
This pull request improves the user interface of the Navigation Drawer in the app. The changes made are :

- Added the Drawer Navigator principal activity in DrawerActivity.java.
- Added fragment classes to include different drawer options. These classes are HomeFragment.java, FilterFragment.java, MyAccountFragment.java, and MyEventFragment.java.
- Added a non-fragment drawer option to the LogInActivity.java class.
- Added the required fragment dependencies in the app build.gradle.
- Added respective .xml files of all fragments, DrawerActivity.java and LogInActivity.java in res/layout folder.
- Added Icons for all options in res/drawable folder.

Looking forward to your review

### CODE REVIEW

File: app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    }
+
+    @After
+    public void tearDown() {
+        Intents.release();
+
+    }
+
+    @Test
+    public void clickDrawerOptionHome_ShowsHomeFragment() {
```
Comment (Lukovitch) [inline app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java:55]: Be careful with the name of the test: lenght and underscore

File: app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
+package com.sdpteam.connectout;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
```
Comment (Lukovitch) [inline app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java:4]: clean the imports please

File: app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java (hunk 1 above)
Comment (ESmike0) [reply app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java:55]: I agree, I will reduce the test name & remove the underscore

File: app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java (hunk 2 above)
Comment (ESmike0) [reply app/src/androidTest/java/com/sdpteam/connectout/DrawerActivityTest.java]: Yes, I agree

**Review by Lukovitch — APPROVED** [review]
Be careful with the unused imports, also check the warning of the CI and more importantly codeclimate for complexity in DrawerActivity.java.

Overall, great job! keep it up

**Pull request merged**